### ISSUE TITLE
Prose and Sections for Website

### ISSUE THREAD

**Issue by candace-walker** [Wesleyan-Media-Project/creative-app#9]
[Prose Document](https://docs.google.com/document/d/1pUIuRYy3ecNpf7lEU0YRLhEzlK0zuXESml873Mn7BdQ/edit#heading=h.e3b32916qeo) @atlasharry and I will add prose to what can be in the section of the [website.](https://www.creativewmp.com) Feel free to check in to the progress, edit, and add. This can also be an area to talk about the type of images to add to the site. 

Investigating adding an updates or releases sections from [embeddings](https://github.com/Wesleyan-Media-Project/fb_2022/tree/master/06_similarity/embeddings) file in the fb_2022 repository. This specific file can find the similarities between ads from their image and text similarity. We could create something in the app for this as well.

**Comment by atlasharry** [comment]
I have created the content for the "Data" section. Here is how I envision it:
![20230725122632](https://github.com/Wesleyan-Media-Project/creative-app/assets/104949958/90dd139b-73f4-420b-8184-3b1b2a503252)

**Comment by SebastianZimmeck** [comment]
Lovely :)

**Comment by candace-walker** [comment]
I have added the [prose](https://docs.google.com/document/d/1pUIuRYy3ecNpf7lEU0YRLhEzlK0zuXESml873Mn7BdQ/edit) for the about section. Let me know what you think and add any corrections needed.

**Comment by sheoftensaid** [comment]
@atlasharry thank you for sharing your data page vision! It might save some time and effort to use a doc so everyone can review/comment/edit on sections/text/layout before trying to design. For example, I am not sure if it makes sense to have the tutorial videos on the data page. If we work in a shared doc instead of images, it will be easier to quickly suggest/make revisions. :)

**Comment by atlasharry** [comment]
> @atlasharry thank you for sharing your data page vision! It might save some time and effort to use a doc so everyone can review/comment/edit on sections/text/layout before trying to design. For example, I am not sure if it makes sense to have the tutorial videos on the data page. If we work in a shared doc instead of images, it will be easier to quickly suggest/make revisions. :)

Thank you for the comment! This definitely makes sense! I will transform my picture into the doc so that everyone can edit it!

**Comment by candace-walker** [comment]
I added the [NSF proposal link ](https://drive.google.com/file/d/1jD7corNzIpNUcsVi2O_2FPy01Lq8wFgT/view?usp=sharing)to the prose document for easy access.

**Comment by candace-walker** [comment]
I made changes to the about challenges section in the about tab and created prose for the team tab.

**Comment by SebastianZimmeck** [comment]
@candace-walker and @atlasharry, is there anything more to do here?

**Comment by candace-walker** [comment]
Yes, there is more to do, but we have taken a break to focus on auditing and repo templates. The overall 1st draft (w/o the visualizations) has been completed, but this shouldn't be closed, yet.

**Comment by SebastianZimmeck** [comment]
As @sheoftensaid [explained](https://github.com/Wesleyan-Media-Project/creative-app/issues/10#issuecomment-1724405347), before this issue can be moved forward, some other points have to be addressed. @sheoftensaid will give the go-ahead once the blocking issues are resolved.

(cc'ing @efranklinfowler)

**Comment by candace-walker** [comment]
Okay, I understand. I will wait for that.
